pages.fr/common/gh-alias.md

```diff
@@ -0,0 +1,24 @@
+# gh alias
+
+> Gérer les alias de commandes GitHub CLI depuis la ligne de commande.
+> Plus d'informations : <https://cli.github.com/manual/gh_alias>.
+
+- Afficher l'aide pour la sous-commande `alias` :
+
+`gh alias`
+
+- Lister tous les alias pour lesquels `gh` est configuré :
+
+`gh alias list`
+
+- Créer un alias de sous-commande pour `gh` :
+
+`gh alias set {{alias}} '{{commande_gh}}`
+
+- Définir une commande shell comme sous-commande de `gh` :
+
+`gh alias set --shell {{nom_de_l'alias}} {{commande}}`
+
+- Supprimer un raccourci de commande :
```

Comment:
```suggestion
- Supprimer un alias :
```

`alias` is fine since you use it line 14. 

also `de commande` imho is not necessary, the sentence sounds more "french" without.

what do you think ?

Reply:
I don't know, the context makes it understandable without it, thus as you like.